x/profiles/client/cli/query.go

```diff
@@ -175,3 +176,53 @@ func GetCmdQueryUserBlocks() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdQueryChainsLinks returns the command allowing to query all the links
+func GetCmdQueryChainsLinks() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "links [user]",
+		Short: "Retrieve all links or links from a particular user by given pagination flags",
+		Args:  cobra.RangeArgs(0, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := types.NewQueryClient(clientCtx)
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			if len(args) == 1 {
+				res, err := queryClient.UserChainsLinks(
+					context.Background(),
+					&types.QueryUserChainsLinksRequest{
+						User:       args[0],
+						Pagination: pageReq,
+					})
+				if err != nil {
+					return err
+				}
+				return clientCtx.PrintProto(res)
+			}
```

Comment:
Maybe here you could use an `else` and avoid to rewrite the following code twice
```go
if err != nil {
       return err
}				
return clientCtx.PrintProto(res)
```

Reply:
The `UserChainsLinks` query function will be removed.